### PR TITLE
Gh/deleted entities

### PR DESCRIPTION
**Pull request by geneh**

### CODE REVIEW

File: providers/fetcher/githubProcessor.js [hunk 1]
```diff
@@ -835,7 +835,7 @@ class GitHubProcessor {
     context.qualifier = qualifier;
     const separator = qualifier.endsWith(':') ? '' : ':';
     request.linkResource(name, `${qualifier}${separator}${type}:${payload[name].id}`);
-    const newContext = extend(true, {}, { history: request.context.history, qualifier: qualifier });
+    const newContext = extend(true, context, { history: request.context.history, qualifier: qualifier });
```
Comment (jeffmcaffer) [inline providers/fetcher/githubProcessor.js]: const newContext = extend(true, {}, context, { history: request.context.history, qualifier: qualifier });

File: providers/fetcher/githubProcessor.js [hunk 2]
```diff
@@ -849,7 +849,7 @@ class GitHubProcessor {
     context.qualifier = qualifier;
     const separator = qualifier.endsWith(':') ? '' : ':';
     request.linkResource(name, `${qualifier}${separator}${type}:${id}`);
-    const newContext = extend(true, {}, { history: request.context.history, qualifier: qualifier });
+    const newContext = extend(true, context, { history: request.context.history, qualifier: qualifier });
```
Comment (jeffmcaffer) [inline providers/fetcher/githubProcessor.js]: const newContext = extend(true, {}, context, { history: request.context.history, qualifier: qualifier });

**Comment by jeffmcaffer** [comment]
Sigh, I accidentally hit the sync button in this branch and pushed another set of changes.  We should revert commit `b46222f`.  I'm not 100% confident in doing that so want to either sit with someone while I do it or have someone else do it. Sorry.

**Comment by geneh** [comment]
b46222f changes seem identical to a73704a. Not sure if anything should be done here.

**Comment by jeffmcaffer** [comment]
I should have compared.  There are differences.  https://github.com/Microsoft/ghcrawler/compare/a73704a...b46222f

Having said that, the differences are good so we should leave the new commit.

**Pull request merged**